editor/code_editor.cpp

```diff
@@ -142,6 +142,7 @@ void FindReplaceBar::unhandled_input(const Ref<InputEvent> &p_event) {
 
 bool FindReplaceBar::_search(uint32_t p_flags, int p_from_line, int p_from_col) {
 	String text = get_search_text();
+
 	Point2i pos = text_editor->search(text, p_flags, p_from_line, p_from_col);
 
 	if (pos.x != -1) {
```

Comment:
Should we update `results_count_to_current` in the else to `0`?

Reply:
Ummmm maybe this?
```cpp
tring searched = get_search_text();
if (searched.is_empty()) {
	results_count = -1;
	results_count_to_current = -1;
	return;
}
```

or

```cpp
if (search_text->get_text().is_empty() || results_count == -1) {
```
to
```cpp
if (search_text->get_text().is_empty() || results_count > 0) {
```